registry/src/api/grpc.rs

```diff
@@ -110,10 +110,9 @@ impl Registry for Context {
             .await
             .map_err(|_| Status::internal("Something went wrong on our side, sorry :("))?;
 
-        let package_type_raw_value = metadata.r#type;
-        let package_type = PackageType::try_from(package_type_raw_value).map_err(|_| {
-            Status::invalid_argument(format!("Unknown Package Type: {}", package_type_raw_value))
-        })?;
+        let package_type_raw_value = crate::proto::buffrs::package::Type::try_from(metadata.r#type)
+            .expect("Shouldn't happen");
```

Comment:
Also don't use expect! (Except in tests)

In functions that return a `Result` you can always use `.map_err` or `.wrap_err` (from eyre / miette) to map this to an error type that you can propagate with `?`